lib/vauth/ntlm.c

```diff
@@ -591,21 +600,14 @@ CURLcode Curl_auth_create_ntlm_type3_message(struct Curl_easy *data,
       return result;
 
     ptr_ntresp = ntlmv2resp;
-  }
-  else
-#endif
-
-#if defined(USE_NTRESPONSES) && defined(USE_NTLM2SESSION)
-
-#define CURL_MD5_DIGEST_LENGTH 16 /* fixed size */
-
-  /* We don't support NTLM2 if we don't have USE_NTRESPONSES */
-  if(ntlm->flags & NTLMFLAG_NEGOTIATE_NTLM_KEY) {
+# else /* defined(USE_NTLM_V2) */
     unsigned char ntbuffer[0x18];
     unsigned char tmp[0x18];
-    unsigned char md5sum[CURL_MD5_DIGEST_LENGTH];
+    unsigned char md5sum[MD5_DIGEST_LENGTH];
```

Comment:
Why this change? We provide our own define to make sure it exists under a single name. There's no guarantee there's a `MD5_DIGEST_LENGTH` define provided in systems! See d80d419d3e that introduced our name.

Reply:
You're right, thanks for pointing it.
I intended to replace by `MD5_DIGEST_LEN` which is defined in `curl_md5.h`. Will fix it.

Reply:
Fixed by using `MD5_DIGEST_LEN`.

Reply:
Do you remove the `CURL_MD5_DIGEST_LENGTH` define as well then?

Reply:
Yes, because it was local to vauth/ntlm.c.